.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 /test-results/
 /playwright-report/
 /playwright/.cache/
+.idea/
```

Comment:
@VaishSiddharth is this change needed?

Reply:
Yes I added this otherwise `.idea` folder might get pushed to github.
`.idea` folder is created by IDE (IntelliJ IDEA)